js/packages/web/src/contexts/tokenList.tsx

```diff
@@ -34,7 +34,7 @@ export function SPLTokenListProvider({ children = null as any }) {
     ]: [WRAPPED_SOL_MINT]
 
   useEffect(() => {
-    getTokenListContainerPromise().then(setTokenList);
+    getTokenListContainerPromise().then(()=>setTokenList);
```

Comment:
Hmm… the old code used to _run_ the `setTokenList` function, but the new code just _returns_ it without running. I’m pretty sure this means the token list is no longer being set.

I haven’t looked yet, what was the build error?